composer.json

```diff
@@ -29,6 +29,7 @@
 		"asika/simple-console": "^1.0",
 		"bacon/bacon-qr-code": "^2.0.0",
 		"divineomega/password_exposed": "^2.8",
+                "enyo/dropzone": "^5.9",
```

Comment:
Please follow the indentation convention for JSON files with 2 spaces.
```suggestion
		"enyo/dropzone": "^5.9",
```

Reply:
sorry... my vimrc gets better and better :)